argus/config/openstack.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, config):
         super(OpenStackOptions, self).__init__(config, group="openstack")
         self._options = [
             cfg.StrOpt(
-                "image_ref", default=None, required=True,
```

Comment:
Because it's required we need to pass some "fake" date in order to start the Argus run, even if we don't use the OpenStack backend some date needs to in order to parse all the config options. But this should not be changed for now.